CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Upcoming
 
+### Fixes
+* Fix `check_subject_age` to parse years. [PR #314](https://github.com/NeurodataWithoutBorders/nwbinspector/pull/314)
```

Comment:
```suggestion
* Fix `check_subject_proper_age_range` to parse years. [PR #314](https://github.com/NeurodataWithoutBorders/nwbinspector/pull/314)
```

`check_subject_age` only checks for ISO validity, no parsing for value comparisons